Code/autopkgserver/packager.py

```diff
@@ -358,7 +358,11 @@ def verify_relative_valid_path(root, path):
                      str(entry.user),
                      str(entry.group)))
 
-            chownpath = os.path.join(self.tmp_pkgroot, entry.path)
+            if entry.path.startswith('/'):
+                chownpath = os.path.join(self.tmp_pkgroot,
+                    entry.path.lstrip('/'))
+            else:
+                chownpath = os.path.join(self.tmp_pkgroot, entry.path)
```

Comment:
Why not just the simpler:

`chownpath = os.path.join(self.tmp_pkgroot, entry.path.lstrip('/'))`

to replace line 361 and not have all the conditional stuff?  Are there any other os.path.join calls that might have similar issues?